05_syntax_lab/01.py

```diff
@@ -1,5 +1,8 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+print "Please write 10 numbers"
+
+high = int(raw_input())
+for i in range (1,10):
+	number = int(raw_input())

```

Comment:
Could you include a prompt message that will tell the user to type in the i-th number?